cayenne/src/test/java/org/apache/cayenne/access/DataContextFlattenedAttributesIT.java

```diff
@@ -457,28 +457,42 @@ public void testDelete() throws Exception {
         a.setArtistName("AX");
         context.commitChanges();
 
-        CompoundPainting o1 = context.newObject(CompoundPainting.class);
-        o1.setArtistName("A1");
-        o1.setEstimatedPrice(new BigDecimal(1.0d));
-        o1.setGalleryName("G1");
-        o1.setPaintingTitle("P1");
-        o1.setTextReview("T1");
+        createTestDataSet();
```

Comment:
I think it's better to keep an old test as is and just add a new one for the task

Reply:
I replaced it because the old test was faulty. If you prefer I can restore it with corrections though ?

Reply:
Yeah, I think we should fix it. There's always shortage of the tests for corner cases.